tests/test_averaging.py

```diff
@@ -117,10 +117,29 @@ def test_allreduce_grid():
     dht.shutdown()
 
 
+@pytest.mark.forked
+def test_allreduce_client_ugly_please_replace():
```

Comment:
Change the name with something more adequate (better yet, make sure that the test is complete before merging)